cosmo_tester/test_suites/cli/__init__.py

```diff
@@ -57,6 +62,101 @@ def _test_cfy_install(run, example, paths, logger):
     example.check_files()
 
 
+def _set_ssh_in_profile(run, example, paths):
+    run(
+        '{cfy} profiles set --ssh-user {ssh_user} --ssh-key {ssh_key}'.format(
+            cfy=paths['cfy'],
+            ssh_user=example.manager.username,
+            ssh_key=paths['ssh_key'],
+        )
+    )
+
+
+def _test_cfy_logs(run, cli_host, example, paths, logger):
+    _set_ssh_in_profile(run, example, paths)
+
+    # stop manager services so the logs won't change during the test
+    example.manager.run_command('cfy_manager stop')
+
+    logs_dump_filepath = [v for v in json.loads(run(
+        '{cfy} logs download --json'.format(cfy=paths['cfy'])
+    ).stdout.strip())['archive paths']['manager'].values()][0]
+
+    log_hashes = [f.split('  ')[0] for f in example.manager.run_command(
```

Comment:
I think you can just do f.split()[0] - if no separator is supplied then it splits on any whitespace and removes empty strings from the result.